core/lib/dal/src/tee_proof_generation_dal.rs

```diff
@@ -195,11 +190,76 @@ impl TeeProofGenerationDal<'_, '_> {
             .await?;
         if result.rows_affected() == 0 {
             let err = instrumentation.constraint_error(anyhow::anyhow!(
-                "Unable to insert TEE attestation: given pubkey already has an attestation assigned"
+                "Unable to insert TEE attestation: pubkey {:?} already has an attestation assigned",
+                pubkey
             ));
             return Err(err);
```

Comment:
I understand that it's not a part of this PR, but same here -- is it a hard error if we will have double attestation? For example, imagine that we receive an attestation request, we save it, but our response is dropped. The TEE machine will retry attestation, and it would cause an error because we've already saved it.

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/2474/commits/3aa30b230794a2a02721abfce63ac5fc69579273). ✅